buildWeb Failed

Hi , 
I go this issue while installing , 

Command :  gradlew distZip

Starting a Gradle Daemon (subsequent builds will be faster)
> Task :buildWeb FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':buildWeb'.
> A problem occurred starting process 'command './deploy.sh''

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 56s
1 actionable task: 1 executed

Hi, Thanks for responding. Upadated node-sass to 4.14.1 . still getting the issue . 

Does this work  in windows ?

> Task :buildWeb FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':buildWeb'.
> A problem occurred starting process 'command './deploy.sh''

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
1 actionable task: 1 executed

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -13,3 +13,25 @@ jobs:
       - name: Gradle Checks
         run: |
           ./gradlew check
+  npm:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        working-directory: web
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+        architecture:
+          - x64
+    name: ${{ matrix.architecture }} on ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+      - run: npm install
+      - run: npm run build

--- a/web/package.json
+++ b/web/package.json
@@ -57,7 +57,7 @@
     "friendly-errors-webpack-plugin": "^1.6.1",
     "html-webpack-plugin": "^2.30.1",
     "node-notifier": "^5.1.2",
-    "node-sass": "^4.9.2",
+    "node-sass": "^4.14.1",
     "optimize-css-assets-webpack-plugin": "^3.2.0",
     "ora": "^1.2.0",
     "portfinder": "^1.0.13",

